src/Function.cpp

```diff
@@ -911,6 +911,21 @@ class SubstituteCalls : public IRMutator {
         : substitutions(substitutions) {}
 };
 
+class SubstituteScheduleParamExprs : public IRMutator {
+    using IRMutator::visit;
+
+    void visit(const Variable *v) override {
+        IRMutator::visit(v);
+        if (v->param.defined() && v->param.is_removed_before_lowering()) {
```

Comment:
What do you think about is_bound_before_lowering instead of is_removed_before_lowering?